### PR TITLE
feat(TDP-1942): Integrate Times Components to NewsKit API

### PR DESCRIPTION
**Pull request by DMackintosh-TNL**
### Description
To support the implementation of “Today’s `<section>`" rail driven by the Product Platforms ‘Recommendations Engine’, integrate Times Components with the NewsKit API.

[TDP-1942](https://nidigitalsolutions.jira.com/browse/TDP-1942)

### Checklist
- [x] Have you done any manual testing?
- [ ] Does it have automated tests?
- [ ] Do you need any other PRs merged before this (if so please list)?
- [ ] Do you need to update the README/Runbook
- [ ] Have you checked for [accessibility](https://nidigitalsolutions.jira.com/wiki/spaces/TNLDigital/pages/3898048523/Accessibility+Dev+Guide#Checklist)?

### Screenshots (if appropriate):
Screenshot 2022-05-20 at 17 03 07

### CODE REVIEW

File: packages/provider-queries/src/recommended_articles.graphql [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        children {
+          __typename
+        }
+      }
+      template
+      updatedDateTime
+      url
+    }
+  }
+}
```
Comment (SunshineEleda) [inline packages/provider-queries/src/recommended_articles.graphql:36]: I've been working on the ui ticket and based on what I saw in the screenshot we only need the image, id, shortidentifier, headline, url, slug. So I'm not sure we will need all of the other information. At the moment just missing the shortidentifier from your query

File: packages/provider/src/recommended-articles.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import { recommendations } from "@times-components/provider-queries";
+import connectGraphql from "./connect";
+
+export default connectGraphql(recommendations);
```
Comment (SunshineEleda) [inline packages/provider/src/recommended-articles.js:4]: Can a test be added for this please?

**Review by SunshineEleda — CHANGES_REQUESTED** [review]
Looks good, just left a few comments. Also with my ticket I will override the component you have made.

File: packages/provider-queries/src/recommended_articles.graphql (hunk 1 above)
Comment (DMackintosh-TNL) [reply packages/provider-queries/src/recommended_articles.graphql:36]: No, totally agree, the query retrieves "everything" at the moment as I didn't know what was required, I can re-write it though ;)

File: packages/provider/src/recommended-articles.js (hunk 2 above)
Comment (DMackintosh-TNL) [reply packages/provider/src/recommended-articles.js:4]: Maaaaaaybe... Sure, not a problem..

File: packages/provider-queries/src/recommended_articles.graphql (hunk 1 above)
Comment (DMackintosh-TNL) [reply packages/provider-queries/src/recommended_articles.graphql:36]: Might be worth noting that `shortidentifier` doesn't exist yet, as the work still needs to be done to make the feed better

File: packages/provider/src/recommended-articles.js (hunk 2 above)
Comment (DMackintosh-TNL) [reply packages/provider/src/recommended-articles.js:4]: As discussed, tests no longer required...

**Review by SunshineEleda — APPROVED** [review]

**Review by othetgyi — COMMENTED** [review]
Nice work, Darren! This looks very cleanly done. 

Because of the linting errors, I'm wondering if recommended_articles.graphql needs to have  formatter run on it? Also, to meet the test coverage threshold, do you need a unit test for the RecommendedArticles.tsx component?

**Comment by DMackintosh-TNL** [comment]
> Nice work, Darren! This looks very cleanly done.
> 
> Because of the linting errors, I'm wondering if recommended_articles.graphql needs to have formatter run on it? Also, to meet the test coverage threshold, do you need a unit test for the RecommendedArticles.tsx component?

I'll take a look ;)

**Review by othetgyi — APPROVED** [review]

**Pull request merged**